api/configuration/v1/kongplugin_types.go

```diff
@@ -17,10 +17,9 @@ limitations under the License.
 package v1
 
 import (
+	"github.com/kong/go-kong/kong"
```

Comment:
Any particular reason to change this?

Reply:
`make lint` did that.

Reply:
`github.com/kong/go-kong` is not treated differently than `default`. I'm not sure we want to change this. https://github.com/Kong/kubernetes-configuration/blob/731c042fb18434ffac0144f222c6f0f90258fb0a/.golangci.yaml#L39-L42

Reply:
Ok let's leave it at that. I think that adding it as a separate group could introduce import block that's too segmented.